packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -420,6 +438,24 @@ class DescopeWc extends BaseDescopeWc {
       this.#handleSdkResponse(sdkResp);
     }
 
+    if (action === RESPONSE_ACTIONS.native) {
+      // prepare a callback to receive the response and failure from the Android layer
+      this.nativeComplete = async (input: any) => {
+        const sdkResp = await this.sdk.flow.next(
```

Comment:
Where/when do we call this fn?

Reply:
The native layer will call this function when it is finished processing the payload received from the previous response